mender-convert-modify

```diff
@@ -205,8 +205,11 @@ run_and_log_cmd "echo 'device_type=${device_type}' > work/device_type"
 run_and_log_cmd "sudo install -m 0444 work/device_type work/rootfs/data/mender/"
 
 if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
-    run_and_log_cmd "echo 'artifact_name=${MENDER_ARTIFACT_NAME}' \
-                     | sudo tee work/rootfs/etc/mender/artifact_info"
+    log_info "Installing the bootstrap Artifact"
+    run_and_log_cmd "mender-artifact write bootstrap-artifact \
+                        --artifact-name "${MENDER_ARTIFACT_NAME}" \
+                        --device-type  "${device_type}" \
+                        --output-path work/rootfs/data/mender/bootstrap.mender"
```

Comment:
AFAIK, the bootstrap artifact doesn't provide any data of its own. I believe this was done so that users who want to deploy special bootstrap artifacts can start with a clean slate. So you will need to add this in the form of `-p rootfs-image.version:NAME` and `-p rootfs-image.checksum:abc...`. Check what a normal rootfs-image artifact has, just to be sure.

Reply:
That answers my question too.